samples/StaticHtmlComposites/playwright.config.ts

```diff
@@ -1,24 +0,0 @@
-// playwright.config.ts
```

Comment:
intentionally deleted?

Reply:
Yes: https://github.com/Azure/communication-ui-library/pull/1252/files?authenticity_token=8apjI8uIvIx6QwC7W4PKz%2F8mXyx9YQ08zWJROjnC8Ss1bZpWYAul2KyJH62blWMX6urx1xHdKS1nUGU0nQWTEQ%3D%3D&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.ts&file-filters%5B%5D=.yaml#diff-a7e4059ab620c34edf008f471b5543bfdd41b0da2f2c663c7a327485b1048818L1

Reply:
Ah this was an obsolete file now it exists in `tests/` directory